docs/diagrams/DeleteSequenceDiagram.puml

```diff
@@ -37,14 +37,14 @@ activate DeleteCommand
 DeleteCommand --> DeleteCommandParser :
 deactivate DeleteCommand
 
-DeleteCommandParser --> AddressBookParser : d
+DeleteCommandParser --> AcademyAssistParser : d
 deactivate DeleteCommandParser
 'Hidden arrow to position the destroy marker below the end of the activation bar.
-DeleteCommandParser -[hidden]-> AddressBookParser
+DeleteCommandParser -[hidden]-> AcademyAssistParser
 destroy DeleteCommandParser
 
-AddressBookParser --> LogicManager : d
-deactivate AddressBookParser
+AcademyAssistParser --> LogicManager : d
+deactivate AcademyAssistParser
```

Comment:
Thanks for updating the all the variables name from `AddressBook` to `AcademicAssist`!